### ISSUE TITLE
VIVO: Renaming rgKey

### ISSUE THREAD

**Issue by aaryani** [researchgraph/Crosswalks#62]
Given RG is the trademark of Research Gate. I recommend we rename rgKey to graphKey.

**Comment by mconlon17** [comment]
Yes.  Please make the changes in the XSL and resulting XML.  I will make the corresponding changes in the researchgraph ontology in the VIVO demo site.
